diff --git a/build.sbt b/build.sbt
index 59551123e..88eb70276 100644
--- a/build.sbt
+++ b/build.sbt
@@ -50,6 +50,7 @@ def commonSettings(projectName: String): Seq[sbt.Def.Setting[_]] = Seq(
   scalaVersion := `scala-2.13`,
   semanticdbEnabled := true, // enable SemanticDB
   semanticdbVersion := scalafixSemanticdb.revision, // use Scalafix compatible version
+  PB.runProtoc in Compile := (args => Process("@protobuf@/bin/protoc", args)!),
   ThisBuild / scalafixScalaBinaryVersion := CrossVersion.binaryScalaVersion(scalaVersion.value),
   ThisBuild / scalafixDependencies ++= List(
     "com.github.liancheng" %% "organize-imports" % "0.5.0",
